implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpConfig.java

```diff
@@ -87,6 +87,19 @@ default Map<String, String> resourceAttributes() {
         return resourceAttributes;
     }
 
+    /**
+     * {@link AggregationTemporality} of the OtlpMeterRegistry. This determines whether
+     * the meters should be cumulative(AGGREGATION_TEMPORALITY_CUMULATIVE) or
+     * step(AGGREGATION_TEMPORALITY_DELTA).
+     * @return the aggregationTemporality for OtlpRegistry
+     * @see <a href=
+     * "https://opentelemetry.io/docs/reference/specification/metrics/data-model/#temporality">Temporality</a>
+     */
+    default AggregationTemporality aggregationTemporality() {
+        return getEnum(this, AggregationTemporality.class, "aggregationTemporality")
+                .orElse(AggregationTemporality.AGGREGATION_TEMPORALITY_CUMULATIVE);
```

Comment:
It seems like it may be better to create our own enum that we can map to the OTLP proto enum (if need be) rather than directly referencing it here. Their enum has 4 values but I think we probably only want to allow cumulative or delta for now. What would we do if this configuration is set to `UNRECOGNIZED`? Does it make sense to allow configuring `AGGREGATION_TEMPORALITY_UNSPECIFIED`?

Reply:
[OTLP Proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L247) says that,

> UNSPECIFIED is the default AggregationTemporality, it MUST not be used.
Honestly, not sure how to interpret it.  But, I assume that clients should not(it says MUST not🙂) use that???

Coming to having our own Enum seems not a bad idea. It allows external frameworks to avoid introducing additional dependencies.